### ISSUE TITLE
pool layer occured problem "stride > window" after convert tf model to onnx 

### ISSUE THREAD

**Issue by ArtyZe** [microsoft/MMdnn#783]
Platform (like ubuntu 16.04/win10): win10 Anaconda2

Python version:3.5

Source framework with version (like Tensorflow 1.4.1 with GPU): 1.14 with cpu

Destination framework with version (like CNTK 2.3 with GPU): onnx

Pre-trained model path (webpath or webdisk path): imagenet_resnet_v2_50.pb
 
Running scripts:
model_onnx = onnx.load('mmdnn_resnet_v2_50.onnx')
tf_rep = oonx_tf.backend.prepare(model_onnx)

but when i run the command, i got such error in pool op:
strides > window_shape not supported dur to inconsistency between CPU and GPU implementation.
I guess maybe the problem is in pooling layer padding
